src/main/kotlin/lotto/Application.kt

```diff
@@ -0,0 +1,34 @@
+package lotto
+
+fun main() {
+    val amountOfMoney = retryable { InputView.getPurchaseAmount() }
+    val printer = LottoPrinter(amountOfMoney)
+
+    ResultView.displayNumberOfTickets(printer.amountOfTicket)
+    ResultView.displayTickets(printer.bundleOfLottoTicket)
+
+    val lastWeekWinningNumbers = retryable { InputView.getLastWeekWinningNumbers() }
+    val bonusNumber = retryable { InputView.getBonusNumber() }
+    println()
+
+    val machine = LottoMachine(amountOfMoney, lastWeekWinningNumbers, bonusNumber)
+    machine.bundleOfLottoTicket = printer.bundleOfLottoTicket
+    machine.writeResultTable()
+
+    val winStats = machine.winStat
+    winStats.calculateWinningMoney(machine.resultTable)
```

Comment:
Seeing `calculate~`...

Instead of storing the result in an internal variable and retrieving it later, there's a way to return the result immediately and pass it naturally within the block. 🙂

From a state management and side-effect perspective, I find the latter approach better. What are your thoughts?